docker-java-core/src/main/java/com/github/dockerjava/core/DefaultDockerClientConfig.java

```diff
@@ -412,6 +417,10 @@ public final Builder withDockerTlsVerify(Boolean dockerTlsVerify) {
             return this;
         }
 
+        public final boolean isDockerHostSetExplicit() {
```

Comment:
@rnorth I need your help with naming 😅 shouldn't it be "explicitly"?

Reply:
uff, it probably should :D